atlasdb-commons/src/main/java/com/palantir/util/AssertUtils.java

```diff
@@ -65,7 +65,7 @@ public static void assertAndLog(boolean cheapTest, String format, Object... args
 
     public static void assertAndLogWithException(boolean cheapTest, String msg, Throwable t) {
         if (!cheapTest) {
-            LoggerFactory.getLogger(AssertUtils.class).error(msg, t);
+            LoggerFactory.getLogger(AssertUtils.class).error("Assertion {} with exception ", msg, t);
             assert false : msg;
```

Comment:
I think this will be captured by internal product testing where ``-ea`` flag is used. Also, this is true for a number of ``assert`` statements in the codebase.